qdmr_parsing/model/run_model.py

```diff
@@ -4,12 +4,19 @@
 import os
 import pandas as pd
 import random
+import sys
+from copy import deepcopy
 
+sys.path.append('.')
 from evaluation.decomposition import Decomposition, get_decomposition_from_tokens
 from model.rule_based.rule_based_model import RuleBasedModel
 from model.rule_based.copy_model import CopyModel
 from model.seq2seq.seq2seq_model import Seq2seqModel
 
+sys.path.append('..')
+from annotation_pipeline.utils.app_store_generation import valid_annotation_tokens
```

Comment:
Here it is an issue, because the code for getting the valid tokens is located outside of the `qdmr_parsing` root directory. Please let me check with @tomerwolgithub.

Reply:
Although I don't like this solution, that's probably the simplest solution to have at this point.
Ideally, we should have had a unified root directory or 3 separated repositories with one being used as a subpackage of another. But that's a too big modification for such a small dependency.
FYI @tomerwolgithub .

@kansalaman Please just add this comment above these lines:
> That's a workaround to solve this small dependency between the root directories "qdmr_parsing" and "annotation_pipeline", which have been developed separately but are under the same GitHub repository.